packages/cosmic-swingset/src/chain-main.js

```diff
@@ -46,7 +46,7 @@ const makeChainStorage = (call, prefix = '', options = {}) => {
     prefix.endsWith('.') ||
     Fail`prefix ${prefix} must end with a dot`;
 
-  const { fromChainShape, toChainShape } = options;
+  const { fromChainShape, toChainShape, notify = true } = options;
```

Comment:
I've always found boolean options defaulting to `true` to be confusing. We can keep it here, but let's at least be explicit when building the mailbox chain storage and pass `notify: true` on line 249.

Reply:
Fixed, where `setterMethod = 'set'` and that is the notification path.